Documentation/Getting-Started-For-Windows.md

```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```

Comment:
This didn't used to be necessary; coreclr build would pull in pre-built corefx bits with nuget.
@mmitche / @adiaaida / @richardlford, has something changed that makes it necessary to build corefx locally?  Do our CI jobs build corefx locally now, or still use nuget?
@alxest, what was the first error you got when you omitted this line?